plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -71,6 +72,7 @@ public static DistributedQueryRunner createIcebergQueryRunner(
     {
         Session session = testSessionBuilder()
                 .setCatalog(ICEBERG_CATALOG)
+                .setSystemProperty(SCALE_WRITERS, "false")
```

Comment:
Why do we want to keep it off by default for Iceberg?

Reply:
If you change it to `true` or comment out this line, the test fails. My PR enables `scaleWriters` by default. It looks like some tests in `IcebergQueryRunner` depends on the fact that `scaleWriters` was disabled by default before my change.

Reply:
How many tests are failing? How difficult do you think it would be to fix these tests?

Reply:
If I remember correctly, 3 or 4 tests fails. To fix it "blindly" it is easy - just change the value in the assertion. But, to be ensured that assertions is valid (it sould be in fact because my change does not influence on scaleWriters itself)- it is not so easy for me at that moment. If you think that it is necessary, I can do it.

Reply:
Yeah, I aggree, it can probably be done as a follow up. What do you think about setting the `setSystemProperty(SCALE_WRITERS, "false")` for those 3 or 4 tests specifically and leaving a comment?

Reply:
I moved setting `SCALE_WRITERS` to `false` to `BaseIcebergConnectorTest` as extra property. I think that it is proper place for this setting, what do you think?

Reply:
I was trying to suggest disabling scaled writers only for the tests that fail with scaled writers enabled. This way we make sure the new tests added are executed with the default settings and it will also help us explicitly identify the tests that don't pass.

Reply:
Thanks for your reply. I understand what do you mean :) 
But, there is no problem in test code specifically. Tests fail because of the initialisation code - the `scaleWriters` option has impact only for method that prepares the environment before actual executing tests. This is why I needed to place “configure” property for `queryRunner` in `BaseIcebergConnectorTest`.  

If it is not still ok, I can fix specific tests by fixing them itself but it is not possible to add this property on the test method level (obviously, it is possible but it makes some mess - it would better to just rewrite testcase.